docs/api-gateway/06-01-api.md

```diff
@@ -53,6 +60,32 @@ These are the possible scenarios depending on the values the the **spec.authenti
 - If you set the value to `true` and the **spec.authentication** parameter value is not empty, authentication is enabled for the service. Entries from the **spec.authentication** parameter are used to create the value of the **origins** attribute of the corresponding Istio Authentication Policy.
 - If you set the value to `true` and the **spec.authentication** parameter value is empty or not provided, authentication is enabled for the service. The default authentication settings specified in API Controller environment variables are used to create the value of the **origins** attribute of the corresponding Istio Authentication Policy.
 
+### Configuration of resources excluded from JWT validation
```

Comment:
Please remove this section, I added it to `03-01` instead.

Reply:
Done.